lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ClockService.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import java.time.Clock;
+import java.time.temporal.ChronoUnit;
+import java.util.Objects;
+
+import static uk.gov.di.ipv.cri.kbv.api.config.ConfigurationConstants.SESSION_TTL;
+
+public class ClockService {
```

Comment:
This class isn't a `ClockService` as such
It is more of a configuration-type class which is providing the session time-to-live
It would make sense for this class to be removed and to move the logic to the existing ConfigurationService